src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -871,9 +885,11 @@ UCS_CLASS_INIT_FUNC(uct_rc_mlx5_iface_common_t, uct_iface_ops_t *tl_ops,
         goto cleanup_dm;
     }
 
-    status = uct_rc_mlx5_devx_iface_init_events(self);
-    if (status != UCS_OK) {
-        goto cleanup_mpool;
+    if (md->flags & UCT_IB_MLX5_MD_FLAG_DEVX) {
```

Comment:
Given the test failures, need to set these in 'else' part - instead of in uct_rc_mlx5_devx_iface_init_events():

    iface->event_channel    = NULL;
    iface->cq_event_channel = NULL;

Reply:
Done. Waiting for CI test result.